benchmarks/prune_test.go

```diff
@@ -56,37 +56,36 @@ func runBlockChain(b *testing.B, prefix string, keepEvery int64, keepRecent int6
 		if err != nil {
 			b.Errorf("Can't save version %d: %v", i, err)
 		}
-		// // Pause timer to garbage-collect and remeasure memory usage
-		// b.StopTimer()
-		// runtime.GC()
-		// runtime.ReadMemStats(&mem)
-		// // update memSize if it has increased after saveVersion
-		// if memSize < mem.Alloc {
-		// 	memSize = mem.Alloc
-		// 	maxVersion = i
-		// }
-		// b.StartTimer()
+
+		// Pause timer to garbage-collect and remeasure memory usage
 		b.StopTimer()
 		runtime.GC()
+		runtime.ReadMemStats(&mem)
+		// update memSize if it has increased after saveVersion
+		if memSize < mem.Alloc {
+			memSize = mem.Alloc
+			maxVersion = i
+		}
 		b.StartTimer()
 	}
+
```

Comment:
Add comment to pause timer so we don't measure db-closing time

Also, remove print statement comment